packages/datatrak-web/src/AppProviders.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React, { ReactNode } from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { ThemeProvider as MuiThemeProvider, StylesProvider } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider } from 'styled-components';
+import { theme } from './theme';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 1000 * 60 * 5, // 5 minutes
+      refetchOnWindowFocus: false,
+      retry: 1,
+      keepPreviousData: true,
```

Comment:
```suggestion
      retry: 0,
      keepPreviousData: false,
```
I'm not sure about keepPreviousData in this app. Maybe we can leave it as the default which is false and see how it goes.